dexs/saucerswap/index.ts

```diff
@@ -1,45 +1,57 @@
 import { ChainBlocks, FetchOptions, SimpleAdapter } from "../../adapters/types";
 import { CHAIN } from "../../helpers/chains";
-import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
-import { getPrices } from "../../utils/prices";
 import { httpGet } from "../../utils/fetchURL";
 
-const historicalVolumeEndpoint = (to: number) =>`https://server.saucerswap.finance/api/public/stats/platformData?field=VOLUME&interval=DAY&from=1650586&to=${to}`
-// https://server.saucerswap.finance/api/public/stats/platformData?field=VOLUME&interval=DAY&from=1650586&to=1682093355
-interface IVolumeall {
+const methodology = {
+  Fees: "A 0.3% fee is charged to users on every swap, 5/6 goes to liquidity providers, 1/6 goes to protocol.",
+}
+
+const historicalVolumeEndpoint = (to: number) => `https://server.saucerswap.finance/api/public/pools/platform-data?field=VOLUME_USD&interval=DAY&from=1650586&to=${to}`
+// https://server.saucerswap.finance/api/public/pools/platform-data?field=VOLUME_USD&interval=DAY&from=1650586&&to=1743155400
+interface IVolumeItem {
   timestampSeconds: string;
-  valueHbar: string;
+  value: string;
 }
 
 const fetch = async (timestamp: number , _: ChainBlocks, { createBalances, startOfDay }: FetchOptions) => {
-  const historicalVolume: IVolumeall[] = (await httpGet(historicalVolumeEndpoint(new Date().getTime() / 1000), { headers: {
-    'origin': 'https://analytics.saucerswap.finance',
+  const historicalVolume: IVolumeItem[] = (await httpGet(historicalVolumeEndpoint(new Date().getTime() / 1000), { headers: {
+    'origin': 'https://www.saucerswap.finance',
   }}));
 
-  const totalVolume = historicalVolume
-    .filter(volItem => Number(volItem.timestampSeconds) <= startOfDay)
-    .reduce((acc, { valueHbar }) => acc + Number(valueHbar), 0)
-
   const _dailyVolume = historicalVolume
-    .find(dayItem => Number(dayItem.timestampSeconds) === startOfDay)?.valueHbar
+    .find(dayItem => Number(dayItem.timestampSeconds) === startOfDay)?.value
+
+  const _dailyVolumeUsd = Number(_dailyVolume ? _dailyVolume : 0)
 
+  // https://docs.saucerswap.finance/protocol/saucerswap-v1
+  // v1 charges fee 0.3% per swap, 5/6 goes to LP, 1/6 goes to protocol
+  const _dailyFeesUsd = _dailyVolumeUsd * 0.003
+  const _dailyRevenueUsd = _dailyFeesUsd * 1/6
 
   const dailyVolume = createBalances()
-  dailyVolume.addCGToken("hedera-hashgraph", (_dailyVolume as any)/1e8)
+  dailyVolume.addUSDValue(_dailyVolume)
+
+  const dailyFees = createBalances()
+  dailyFees.addUSDValue(_dailyFeesUsd)
+
+  const dailyRevenue = createBalances()
+  dailyRevenue.addUSDValue(_dailyRevenueUsd)
 
   return {
-    // totalVolume: totalVolume ? String(totalVolume/1e8 * prices[coinId].price) : "0",
     dailyVolume,
+    dailyFees,
+    dailyRevenue,
     timestamp: startOfDay,
   };
 };
 
-
 const adapter: SimpleAdapter = {
```

Comment:
except for minor nitpick, the code looks good, can you add a fee adapter that points to this file like here: https://github.com/noateden/defillmana-dimension-adapters/blob/pr/noateden/2726/fees/agdex.ts#L2